packages/ui/src/components/Card/Card.tsx

```diff
@@ -66,4 +66,4 @@ function Meta({ title, description, style, className }: MetaProps) {
 }
 
 Card.Meta = Meta
-export default Card
+export default Card
```

Comment:
Hey @karthiknadar1204, this diff means that the newline was removed at the end of the file. Typically we keep an extra newline at the end of each file. Can we add it back?